src/simulation/dynamics/_GeneralModuleFiles/stateVecIntegrator.h

```diff
@@ -35,8 +35,6 @@ class StateVecIntegrator
     virtual void integrate(double currentTime, double timeStep) = 0; //!< class method
```

Comment:
:small_red_triangle_down: The commit message is too long. "Remove uneeded dynPtr" is sufficient.

Reply:
#$@#$@. I'll go fix.